katas/metal-slug/kevin/CharacterStatus.js

```diff
@@ -0,0 +1,28 @@
+export class CharacterStatus {
```

Comment:
This looks interesting, it's a nice way of having a class that defines life points and attack power, will this have a relationship with soldier class?